packages/@webex/internal-plugin-metrics/src/business-metrics.ts

```
@@ -93,30 +33,80 @@ export default class BusinessMetrics extends GenericMetrics {
+          },
+          metadata,
+        });
+
+        return this.submitEvent({
+          kind: 'buisness-events:business_metrics -> ',
+          name,
+          event: businessEvent,
+        });
+      }
+
       case 'business_ucf':
```

Biome (1.9.4)

[error] Useless case clause.

because the default clause is present:

Unsafe fix: Remove the useless case.


(lint/complexity/noUselessSwitchCase)